### PR TITLE
userhome component

### PR DESCRIPTION
**Pull request by bayanseder**
#28 
create UserHome  component
UserHome style

### CODE REVIEW

File: client/src/components/Layout/UserHome/UserHome.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <div className='big_continer'>
+      <div className='user_profile'>
+        <img src={Profile} />
+        <h2>Bayan_Seder</h2>
+      </div>
+      <div className='continer_div'>
+        <h3 className='events'>Your Events</h3>
+        <div className='event_card'>
+          <div className='title'>
+            <h3>How to destroy your life</h3>
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.js]: Work to get the data from BE

File: client/src/components/Layout/UserHome/UserHome.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            <p>23/2/2022</p>
+          </div>
+          <div className='event_option'>
+            <img src={Delete} alt='delete' />
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.js]: Hmm, These two images should to have event listener later, so try to change img to something else

File: client/src/components/Layout/UserHome/UserHome.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+}
+.user_profile h2{
+    margin-top: 0.8em;
+    color: #4FB2AA;
+}
+.continer_div{
+    margin: 0.5em 1em 1em 1em;
+}
+
+.events{
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.css]: The style should be exactly like Figma

File: client/src/components/Layout/UserHome/UserHome.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.css]: The style should be exactly like Figma

File: client/src/components/Layout/UserHome/UserHome.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  }
+  componentDidMount () {
+       axios.get('/api/user-events')
+      .then(res=>this.setState({events: res.data}))
+      .catch(err => console.log(err))   
+   }
+
+
+  render () {
+    return <div className='big_continer'>
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.js]: Use BEM correctly!

File: server/database/queries/getUserHome.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+const dbconniction = require('../db_connection')
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+const userHome = (callBack) => {
+  const sql = {
+    text: 'SELECT * FROM events WHERE user_id =$1',
+    values: [11]
```
Comment (yaqootturman) [inline server/database/queries/getUserHome.js]: What is the values here?

File: client/src/components/Layout/UserHome/UserHome.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+import Delete from '../../../Images/delete.jpg'
+import Edit from '../../../Images/Edit.png'
+import './UserHome.css'
+import axios from 'axios'
+class UserHome extends Component {
+  state = {
+    events :[]
+  }
+  componentDidMount () {
+       axios.get('/api/user-events')
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.js]: You should get the user events, so send parameters related to the user with the request

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: server/database/queries/getUserHome.js (hunk 6 above)
Comment (bayanseder) [reply server/database/queries/getUserHome.js]: here the values  is the user_id .... we will change it when we build the cookies

File: client/src/components/Layout/UserHome/UserHome.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+      .then(res=>this.setState({events: res.data}))
+      .catch(err => console.log(err))   
+   }
+
+
+  render () {
+    return <div className='component_continer'>
+      <div className='user_profile__div'>
+        <img src={Profile} />
+        <h2>Bayan_Seder</h2>
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.js:23]: Change this static data to what you have in state "user.name"

File: server/controller/index.js [hunk 9]
```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const router = express.Router()
+
+const userHome = require('../database/queries/getUserHome')
+
+router.get('/api/user-events/:id', (req, res) => {
```
Comment (yaqootturman) [inline server/controller/index.js]: split the controllers

File: server/controller/index.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+  const reqbody = req.params.id
```
Comment (yaqootturman) [inline server/controller/index.js]: change the 'reqbody' variable name

File: client/src/components/Layout/UserHome/UserHome.js [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+      </div>
+      <div className='events_continer__div'>
+        <h3 className='events'>Your Events</h3>
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.js:26]: Try to make this exactly as Figma

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/components/Layout/UserHome/UserHome.css [hunk 12]
```diff
@@ -0,0 +1,131 @@
+.component_continer{
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.css]: Change the CSS file to camelCase name style

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/components/Layout/UserHome/UserHome.css (hunk 12 above)
Comment (bayanseder) [reply client/src/components/Layout/UserHome/UserHome.css]: done

**Review by tasneembhiri — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Review by Hanan795 — APPROVED** [review]

**Review by aboodswiti — APPROVED** [review]

**Pull request merged**